src/Mock/getMock.js

```diff
@@ -203,5 +203,77 @@ export const getCollectionMock = {
       status: 400,
       statusText: 'Failure'
     }
+  },
+  getWithOrderByAgeDescending: {
+    path: 'tests/get/users',
+    body: { order: { orderBy: ['age'], orderDirection: 'desc'}},
```

Comment:
How about having all in the same body. Like this: 
`body: { orderBy: 'age', orderDirection: 'desc, limit: 20, filters: [...]}`
As to avoid having objects inside objects

Reply:
Is there any chance of ordering for more than one value?
Something like
```sql
select id, clientName, productName from orders order by clientName, productName;
```

Reply:
@FrankIglesias right now firebase doesn't support ordering by multiple fields

Reply:
this is not the REST way to order using a get, ask someone from backend how they handle this using query params

Reply:
I think it should support object and query params